core-web/libs/dotcms-models/src/lib/dot-experiments-constants.ts

```diff
@@ -209,3 +209,5 @@ export const ReportSummaryLegendByBayesianStatus: Record<BayesianLegendStatus, S
         legend: 'experiments.summary.suggested-winner.no-enough-sessions'
     }
 };
+
+export const CONFIRM_DIALOG_KEY = 'confirmDialog';
```

Comment:
This is a unique key only for Configuration's Dialog? Could be something like, CONFIGURATION_CONFIRM_DIALOG_KEY

Reply:
yea just configuration.. will make that change